addons/main/CfgSettings.hpp

```diff
@@ -4,6 +4,8 @@ class CfgSettings {
             class PREFIX {
                 class dependencies {
                     CBA[] = {"cba_main", REQUIRED_CBA_VERSION, "(true)"};
+
+                    compat_sogpf[] = {"sys_sog", {VERSION_AR}, "isClass (configFile >> 'CfgPatches' >> 'data_f_vietnam')"};
```

Comment:
I think it should be this, but not tested
```suggestion
                    compat_sogpf[] = {"acre_sys_sog", {VERSION_AR}, "isClass (configFile >> 'CfgPatches' >> 'data_f_vietnam')"};
```